cms/static/js/view_pros_cons.js

```diff
@@ -19,4 +19,29 @@ function proConListeners() {
   });
 }
 
-proConListeners();
+// shows button for toggle pros/cons. Needs to grab parentNode as this is what is dn
+function reverseButtonVisibility (el) {
+  removeClasses(el.parentNode, ['dn']);
+  addClasses(el.parentNode, ['db']);
+}
+
+// hides the list of pros and cons
+function reverseListVisibility (el) {
+  removeClasses(el, ['dt']);
+  addClasses(el, ['dn']);
+}
+
+// checks if pros and cons are on the page that's being view, and if it's mobile
+function mobileProsAndCons () {
+  if (selectAll('.pros_cons').length > 0 && isMobile()) {
+    selectAll('.pros_cons').forEach(function (el) {
```

Comment:
Does IE8 support forEach? :thinking:

Reply:
It might not, but it's been reworked so that users on IE8 can view pros and cons without JavaScript. So with this change around, it will default to the pros and cons being visible on IE8, and the JS won't need to run to view them.